cytotable/presets.py

```diff
@@ -317,6 +317,33 @@
         # and modified at runtime as needed
         "CONFIG_JOINS": "",
     },
+    "deepprofiler": {
+        # version specifications using related references
+        "CONFIG_SOURCE_VERSION": {
+            "deepprofiler": "v0.3.1",
+            "cellprofiler": "v4.2.x",
```

Comment:
why specified?

Reply:
Based on my understanding from the input data to DeepProfiler CellProfiler may play a part when it comes to DeepProfiler's output. Since this field is largely information-based only (not used in processing currently) I felt it might be good to add the detail here based on where I thought the data might have come from (I used the dates of the commits for the files in Pycytominer as a basis of understanding what most likely was used).